scalameta/dialects/src/main/scala/scala/meta/dialects/package.scala

```diff
@@ -78,6 +83,7 @@ package object dialects {
     def allowInline = false
     def allowSpliceUnderscore = Scala210.allowSpliceUnderscore
     def allowToplevelTerms = true
+    def allowViewBounds = true
```

Comment:
Please reference `Scala210.allowViewBounds` in all dialects where the setting is true.